wrench/src/yaml_frame_writer.rs

```diff
@@ -179,14 +179,10 @@ fn maybe_radius_yaml(radius: &BorderRadius) -> Option<Yaml> {
     }
 }
 
-fn write_sc(parent: &mut Table, sc: &StackingContext, filter_iter: AuxIter<FilterOp>) {
+fn write_sc(parent: &mut Table, sc: &StackingContext, scene: &Scene, filter_iter: AuxIter<FilterOp>) {
```

Comment:
nit: looks like we actually only need the properties here and below, not the whole scene